Readme.md

```diff
@@ -3,65 +3,71 @@
 
-* Install VS Code
-* Open Extensions View (Ctrl+Shift+X)
-* Search for "Camel"
-* Select the "wsdl2rest for Visual Studio Code" entry and click Install
-* Profit!
+After you install VS Code, follow these steps:
+1. In VS Code, select **View > Extensions**.
+2. Search for **Camel**.
+3. Select the **Extension Pack for Apache Camel** or the **WSDL to Camel Rest DSL** option and then click **Install**.
 
-## Using the extension
-There are two main options for using the extension to generate your Camel configuration. To start the process, inside VS Code, press `F1` or `Ctrl+Shift+P` to bring up the Command Palette, and type `wsdl2rest`. Then you must choose whether you're going to use a local WSDL file or a WSDL somewhere on the network.
+## Before you begin
+Before you use this **WSDL to Camel Rest DSL** extension, you should create a mavenized Camel- or Fuse-based project, such as one of the following:
+
+* A project that you create with the [Project Initializer by Red Hat VS Code extension](https://marketplace.visualstudio.com/items?itemName=redhat.project-initializer).   
+Note: This extension is included in the [Extension pack for Apache Camel](https://marketplace.visualstudio.com/items?itemName=camel-tooling.apache-camel-extension-pack).
 
-To reference a WSDL file in your local file system:
+* A standalone Camel- or Fuse-based Spring project based on an existing example or a quickstart, for example:  
+https://github.com/apache/camel/tree/master/examples/camel-example-spring  
+or  
+https://github.com/jboss-fuse/quickstarts/tree/master/beginner/camel-cbr
 
-* Select the 'wsdl2rest: Create Camel Rest DSL configuration from local WSDL file' option in the list.
-* In the File dialog that appears, browse to find your WSDL file in the local file system.
+* A Maven project created by using [mvn:archetype](http://camel.apache.org/camel-maven-archetypes.html) with camel-archetype-spring, camel-archetype-blueprint, or camel-archetype-spring-boot.
+
+After you create a project, you can then use the **WSDL to Camel Rest DSL** extension to generate the Camel Rest DSL + CXF solution for REST-style access. 
+
+## Using the extension
+To generate your Camel configuration from a WSDL file:
 
-To reference a WSDL located somewhere on the Internet (or Intranet) with a URL:
+1. In VS Code, create a new workspace and then add the Camel- or Fuse-based project folder to the workspace.
 
-* Select the 'wsdl2rest: Create Camel Rest DSL configuration from WSDL file URL' option in the list.
-* In the drop-down that appears, type the URL to the WSDL you wish to access.
+2. Select **View > Command Palette** and then type **WSDL to Camel Rest DSL**.  
```

Comment:
Incorrect - just type "wsdl2rest" -- may have to investigate adding other ways to call this, but for now that's the only way to activate it